src/ScreenCover/index.js

```diff
@@ -1,18 +1,40 @@
-import React, { Fragment } from 'react'
+import React from 'react'
 import propTypes from 'prop-types'
 import css from 'styled-jsx/css'
 import cx from 'classnames'
 
-export const ScreenCover = ({ onClick }) => (
-    <div className="screen-cover" onClick={onClick}>
+const ScreenCover = ({ children, onClick }) => (
+    <div className="screen-cover">
+        <div className="backdrop" onClick={onClick} />
+
+        <div className="children">{children}</div>
```

Comment:
You could extract the backdrop and children into their own components to avoid the classname

```jsx
const Backdrop = ({ onClick }) => (
  <div onClick={onClick}>
    <style jsx>{`
      div {
        height: 100%;
        width: 100%;
        background: rgba(200, 200, 200, 0.6);
      }
    `}</style>
  </div>
)

const Children = ({ children }) => (
  <div>
    <style jsx>{`
      div {
        position: absolute;
        top: 50%;
        left: 50%;
        width: auto;
        height: auto;
        transform: translate(-50% -50%);
      }
    `}</style>
  </div>
)
```

And then:
```jsx
<Backdrop onClick={onClick} />
<Children children={children} />
```

Reply:
6357383